gateway/src/cluster/config.rs

```diff
@@ -93,6 +94,7 @@ pub struct ClusterConfig {
     shard_config: ShardConfig,
     shard_scheme: ShardScheme,
     queue: Arc<Box<dyn Queue>>,
+    resume_data: HashMap<u64, (String, u64)>,
```

Comment:
Maybe this and the getter and setter could have a slightly more concise name like `resume_shards`? Don't know, but `data` isn't very descriptive

Reply:
couldn't really find a better name, shards isn't any more descriptive imo as you are not providing shards either